packages/swapper/src/swappers/utils/filterBuyAssetsBySellAssetId/filterBuyAssetsBySellAssetId.ts

```diff
@@ -23,6 +23,19 @@ const _filterEvmBuyAssetsBySellAssetId = (
   })
 }
 
+const _filterSolanaBuyAssetsBySellAssetId = (
+  { assets, sellAsset }: BuyAssetBySellIdInput,
+  chainIdPredicate: ChainIdPredicate,
+): Asset[] => {
+  // evm only
+  if (!isSvmChainId(sellAsset.chainId)) return []
+
+  return assets.filter(buyAsset => {
+    // evm only AND chain id predicate with no arbitrum nova support for any swappers
+    return isSvmChainId(buyAsset.chainId) && chainIdPredicate(buyAsset.chainId, sellAsset.chainId)
+  })
+}
+
```

Comment:
```suggestion
const _filterSolanaBuyAssetsBySellAssetId = (
  { assets, sellAsset }: BuyAssetBySellIdInput,
  chainIdPredicate: ChainIdPredicate,
): Asset[] => {
  // svm only
  if (!isSvmChainId(sellAsset.chainId)) return []

  return assets.filter(buyAsset => {
    // svm only AND chain id predicate
    return isSvmChainId(buyAsset.chainId) && chainIdPredicate(buyAsset.chainId, sellAsset.chainId)
  })
}
```

Though we should be able to remove this entirely after tackling the suggested changes in `filterBuyAssetsBySellAssetId`

Reply:
Totally removed in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)